web-ui/src/components/admin/users/Users.jsx

```diff
@@ -97,11 +95,13 @@ const Users = () => {
           />
           {isAdmin && (
             <div className="add-member">
-              <Button startIcon={<PersonIcon />} onClick={handleOpen}>
+              <Button startIcon={<PersonIcon />} onClick={()=>handleOpen(true)}>
```

Comment:
This line should be changed back. Handle open doesn't take a parameter, and doing this (inline arrow) will create a new function for every re-rendering.

Reply:
I've altered this function back to the original form.